### DISCUSSION TITLE
Default Value

### DISCUSSION THREAD

**Discussion by havvanur160401046** [serenity-is/Serenity#5324] [Help]
![image](https://user-images.githubusercontent.com/24252615/103197060-69da4900-48f6-11eb-9f8c-049e3b417398.png)

I will add records to the sales table, but I want the staff column to automatically come to the logged in user. How can I do?
example: Logged in user to staff column in 'admin' sales table By default I just want him to come

**Comment by arunkumar12712** [comment]
like this
this.form.CreatedBy.value = Authorization.userDefinition.UserId;

**Reply by havvanur160401046** [reply]
which file should I edit? 
xdialog.ts or  xColumns.cs

**Reply by arunkumar12712** [reply]
xdialog.ts

**Reply by havvanur160401046** [reply]
![image](https://user-images.githubusercontent.com/24252615/103336007-dd5f9000-4a87-11eb-80bf-5ab55a847649.png)
there is no UserId :(

**Reply by havvanur160401046** [reply]
I solved the problem, thanks :)

**Reply by srinivas1910** [reply]
hi can you share the solution

**Comment by JohnRanger** [comment]
@havvanur160401046 ,

depending on your requirements you could also - at record create - simply fill in the current user's id in the onBeforeSave method within the xyzRepository.cs and not showing this field at record create event - only show it when later editing the record.

This works of course only when the user should not be changeable at create event from within the GUI.

With kind regards,

John

**Reply by srinivas1910** [reply]
hi @JohnRanger  
can you please share a sample code for this

With regards,
Srinivas
